DataFormats/SiPixelDigiSoA/interface/alpaka/SiPixelDigisDevice.h

```diff
@@ -0,0 +1,53 @@
+#ifndef DataFormats_SiPixelDigi_interface_SiPixelDigisDevice_h
+#define DataFormats_SiPixelDigi_interface_SiPixelDigisDevice_h
+
+#include <cstdint>
+#include <alpaka/alpaka.hpp>
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "DataFormats/SiPixelDigiSoA/interface/SiPixelDigisHost.h"
+#include "DataFormats/SiPixelDigiSoA/interface/SiPixelDigisLayout.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/CopyToHost.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class SiPixelDigisDevice : public PortableCollection<SiPixelDigisLayout<>> {
+  public:
+    SiPixelDigisDevice() = default;
+    template <typename TQueue>
+    explicit SiPixelDigisDevice(size_t maxFedWords, TQueue queue)
+        : PortableCollection<SiPixelDigisLayout<>>(maxFedWords + 1, queue) {}
+    ~SiPixelDigisDevice() = default;
+
+    SiPixelDigisDevice(SiPixelDigisDevice &&) = default;
+    SiPixelDigisDevice &operator=(SiPixelDigisDevice &&) = default;
```

Comment:
I'd suggest to either rely on compiler declaring these, or add move oprations as well